Ncsdk 18026 bt status service (#9955)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/nsms.rst

.. _nsms_readme:

Nordic Status Message Service (NSMS)
####################################

.. contents::
   :local:
   :depth: 2

> The Nordic Status Message Service (NSMS) is a custom service that provides the name of the status and string describing current status.
> Current status can be notified or can be read to allow status check when device have been just connected.

Reviewer: ```suggestion
The Nordic Status Message Service (NSMS) is a custom service that provides the name of the status and a string describing the current status.
When the device is first connected, the current status can be notified or read to allow for a status check.
```

The NSMS service is used in the :ref:`peripheral_status` sample.

> Multiple instances of the service can be implemented to show status of different part of the system.
> The instances can be distinguished by the client by its name provided in Characteristic User Description (CUD).

Reviewer: ```suggestion
Multiple instances of the service can be implemented to show the status of different parts of the system.
The instances can be distinguished by the client by its name provided in the Characteristic User Description (CUD).
```

.. _nsms_readme:

Nordic Status Message Service (NSMS)
####################################

.. contents::
   :local:
   :depth: 2

The Nordic Status Message Service (NSMS) is a custom service that provides the name of the status and a string describing the current status.
The status can be either notified or directly read (without waiting for notification) to check the current status whenever the device is connected.

The NSMS service is used in the :ref:`peripheral_status` sample.

Multiple instances of the service can be implemented to show the status of different parts of the system.
The instances can be distinguished by the client by its name provided in the Characteristic User Description (CUD).

Service UUID
************

The 128-bit vendor-specific service UUID is 57a70000-9350-11ed-a1eb-x0242ac120002.

Characteristics
***************

This service has one characteristic with has CCC and CUD descriptors.
CUD (Characteristic User Description) holds the name of the Status Message and helps to

> distinguish between different instances of the NSMS on the same device.

Reviewer 2: FYI: The acronym already explained in line 16

.. _nsms_readme:

Nordic Status Message Service (NSMS)
####################################

.. contents::
   :local:
   :depth: 2

The Nordic Status Message Service (NSMS) is a custom service that provides the name of the status and a string describing the current status.

> When the device is first connected, the current status can be notified or read to allow for a status check.

Reviewer 2: ```suggestion
The current status can be either notified or read to allow the status check when the device is first connected.
```

> The current status can be either notified or read to allow the status check when the device is first connected.

Author: ```suggestion
The status can be either notified or directly read (without waiting for notification) to check the current status whenever the device is connected. 
```

The NSMS service is used in the :ref:`peripheral_status` sample.

Multiple instances of the service can be implemented to show the status of different parts of the system.
The instances can be distinguished by the client by its name provided in the Characteristic User Description (CUD).

Service UUID
************

The 128-bit vendor-specific service UUID is 57a70000-9350-11ed-a1eb-x0242ac120002.

Characteristics
***************

> This service has two characteristics.

Author: Proposal:
```
This service has one characteristic with has CCC and CUD descriptors.
CUD (Characteristic User Description) holds the name of the Status Message and helps to
distinguish between different instances of the NSMS on the same device. 
```
